### PR TITLE
Temporarly fix to solve problems with serverinfo on linux.

### PR DESCRIPTION
**Pull request by HalfAway**
Temporarly fix to solve problems with serverinfo on linux.
Made it possible to see loot/magic/skills even if you use stages.

### CODE REVIEW

**Comment by triath** [comment]
Why not just
```
if (file_exists($config['server_path'].'/data/XML/stages.xml')) {
	$stages_path = simplexml_load_file($config['server_path'].'/data/XML/stages.xml');
} elseif (file_exists($config['server_path'].'/data/xml/stages.xml')) {
	$stages_path = simplexml_load_file($config['server_path'].'/data/xml/stages.xml');
} else {
	echo 'Couldn\'t locate stages.xml';
}
```

File: serverinfo.php [hunk 1]
```diff
@@ -5,18 +5,27 @@
...
+	// Checks OS from config.php ($config['os'])
+	if ($config['os'] == 'LINUX') {
+		if (file_exists($config['server_path'].'/data/XML/stages.xml')) {
+			$stages_path = simplexml_load_file($config['server_path'].'/data/XML/stages.xml');
+		} else {
+			echo 'Couldn\'t locate stages.xml';
+		}
+	} else {
+		if (file_exists($config['server_path'].'/data/xml/stages.xml')) {
+			$stages_path = simplexml_load_file($config['server_path'].'/data/XML/stages.xml');
```
Comment (triath) [inline serverinfo.php]: lowercase xml

**Comment by Znote** [comment]
Is that folder name lowercase in any OT distro?

**Comment by triath** [comment]
@Znote Did you catch that merging this PR doesn't actually change anything regarding the path, as the foldername on line 18 needs to be lowercase?

Also I can't see the point of checking the OS instead of just trying both paths as per my previous comment. That way it doesn't matter if the foldername is lowercase or uppercase.

**Comment by Znote** [comment]
Its a bit uneccesary forcing a double file check if only one url path is correct. 

Ohh your right, there seem to still be a lowercase issue in it.

**Pull request merged**